### ISSUE TITLE
Timed out

### ISSUE THREAD

**Issue by gentunian** [Urigo/Meteor-Angular2#44]
Hey there, I was trying to do angular2 tutorial but I got this error:

`$ meteor add urigo:angular2-meteor

 => Errors while adding packages:             
                                              
While downloading urigo:angular2-meteor@0.3.1...:
error: connect ETIMEDOUT
`

Anything wrong?
cheers

**Comment by barbatus** [comment]
Check out your internet connection, it's likely related to the atmospherejs.com rather than to the package itself

**Comment by gentunian** [comment]
Not sure what you think about my internet connection... or what you think about my profile. I do IT so the first thing I checked was that. I'm not a developer.

This happens with this package only. I was doing both tutorials angular 1 and agular 2. Also, I don't see how my internet connection could be wrong.

It would be nice to tell me where I could report this problem if it's not here.

**Comment by barbatus** [comment]
Hey, this package's been installed ~1.5k times, and no one had similar issue (at least no one reported),
what else can I think of except internet connection? I just installed it once more into an empty app, and it worked.
If you still are facing that problem, you can report to where the packages is hosted: [atmospherejs.com](https://github.com/percolatestudio/atmosphere/issues). Please be more self-conscious before submitting issues.

**Comment by gentunian** [comment]
Ok, whatever.

**Comment by Urigo** [comment]
@gentunian it looks like @barbatus is right with his analysis and I don't think he meant to offend you.

Is it still not working for you?

**Comment by gentunian** [comment]
This was not working from home, work and friends. So I don't see how his analysis could be ok when our ISP differs and other packages were installing OK.

I tried now, and seems to work. But the version is gathering it is not the same:

```urigo:angular2-meteor   added, version 0.3.3```

So I'm sure that with 0.3.1 there was a problem somewhere and I apologize if I reported the problem in the wrong side.

Regards
